frontend/src/lib/components/LemonCalendar/LemonCalendar.tsx

```diff
@@ -0,0 +1,141 @@
+import './LemonCalendar.scss'
+import React, { useEffect, useState } from 'react'
+import { dayjs } from 'lib/dayjs'
+import { range } from 'lib/utils'
+import { LemonButton, LemonButtonProps } from 'lib/components/LemonButton'
+import { IconChevronLeft, IconChevronRight } from 'lib/components/icons'
+import clsx from 'clsx'
+
+export interface LemonCalendarProps {
+    /** Fired if a calendar cell is clicked */
+    onDateClick?: (date: string) => void
+    /** YYYY-MM-xx to specify the month that is shown */
+    firstMonth?: string | null
+    /** Called if the user changed the month in the calendar */
+    onFirstMonthChanged?: (date: string) => void
+    /** Use custom LemonButton properties for each date */
+    getLemonButtonProps?: (date: string, month: string, defaultProps: LemonButtonProps) => LemonButtonProps
+    /** Number of months */
+    months?: number
+    /** Number of weeks in each month */
+    weeks?: number
```

Comment:
I suppose this is also cool to play around with… but a month is a month, what is this prop useful for?

Reply:
yeah, removed